zeek_log_transport.sh

```diff
@@ -57,24 +57,22 @@ status () {
 
 
 usage () {
-	echo 'Usage: '"$0"' [--localdir /local/top/dir/] [--dest where_to_ssh] [--remotedir /remote/top/dir/] [--rsyncparams '"' --aparam --anotherparam '"']' >&2
-	echo 'The optional --dest can be a hostname, IP, user@hostname, user@ip, or any label in an ~/.ssh/config stanza' >&2
-	echo 'If left off, we use the "Location" field from /etc/rita/agent.yaml' >&2
-	echo 'The user@... format is discouraged - we want to use dataimport@... on the remote server.' >&2
-	echo '' >&2
-	echo 'The optional --localdir is where the Bro/Zeek logs can be found on this system system.' >&2
-	echo 'If you look in this directory, it should contain at least a directory or symlink called current .' >&2
-	echo 'By default we will look in common locations for this directory tree.' >&2
-	echo '' >&2
-	echo 'The optional --remotedir is where you want the Bro/Zeek logs to end up on the target system.' >&2
-	echo 'If left off, it will be /opt/bro/remotelogs/$my_id/ or /opt/zeek/remotelogs/$my_id/' >&2
-	echo '' >&2
-	echo 'The optional --rsyncparams allows you to specify parameters for rsync.  MAKE SURE to enclose the entire block in a pair of single quotes.  Suggestions:' >&2
-	echo '	--bwlimit=NNN	#Limit bandwidth used to NNN kilobytes/sec' >&2
-	echo '	-v		#Verbose; list out the files being transferred, discouraged if running from cron' >&2
-	echo '	-q		#Turn off any messages that are not errors, encouraged if running from cron' >&2
-	echo '	--dry-run	#Test, do not actually transfer files' >&2
-	echo '	DO NOT add --compress ; the files we are sending are already compressed.' >&2
+	cat <<HEREDOC >&2
+Usage: $0 [--all] [--dest where_to_ssh] [--localdir /local/top/dir/] [--remotedir /remote/top/dir/] [--rsyncparams '--aparam --anotherparam']
+
+Options:
+    --all           Sync all Zeek log types instead of the default subset.
+    --dest          SSH destination target (e.g hostname, IP, user@hostname, user@ip)
+    --localdir      Location of Zeek logs on local system. (default: searches common locations)
+    --remotedir     Location of Zeek logs on remote system. (default: /opt/zeek/remotelogs/<sensorname>/)
+    --rsyncparams   Allows specifying parameters for rsync. Enclose in a pair of single quotes.
+
+        Suggestions:
+            --bwlimit=NNN   Limit bandwidth used to NNN kilobytes/sec
+            -v              Verbose; list out the files being transferred
+            -q              Turn off any messages that are not errors
+            -n              Dry run, do not actually transfer files
+HEREDOC
```

Comment:
I was changing this text to document the new flag anyway so I took a stab at reworking the help text to something more inline with other unix programs. If this change is a problem I can revert to the old help format.